### PR TITLE
[new] add videogular-themes-default@1.4.4

### PR DESCRIPTION
**Pull request by Amomo**
Hi @maruilian11 for #6818

I add  **videogular-themes-default** at v1.4.0 and add `git` auto-update config in its package.json.
repo url: https://github.com/videogular/bower-videogular-themes-default

**videogular-themes-default** is the theme of  **videogular**, I think we could add it.
Would you mind checking this PR for me? Thank you~ :-D

- [x] Not found on cdnjs repo
- [x] No already exist issue and PR
- [x] Notable popularity : 
- **videogular-themes-default** : Watch **`1`**, Star **`1`**, Fork **`12`**
- **videogular** : Watch **`79`**, Star **`1,082`**, Fork **`260`**
- [x] Project has public repository on famous online hosting platform (or been hosted on npm) 
- [x] Has valid tags for each versions (for git auto-update)
- [x] Auto-update setup
- [x] Auto-update target/source is valid.
- [x] Auto-update filemap is correct.

### CODE REVIEW

**Comment by maruilian11** [comment]
＠amomo v1.4.1 seems just released 😹
do we need to host `dev.svg`？

**Comment by Amomo** [comment]
@maruilian11 I have updated it to `v1.4.1`, could you help me checking it again? Thanks! :-D

**Comment by maruilian11** [comment]
@PeterDaveHello i think this PR is ok.

File: ajax/libs/videogular-themes-default/package.json [hunk 1]
```diff
@@ -0,0 +1,26 @@
...
+  "repository": {
+    "type": "git",
+    "url": "https://github.com/videogular/bower-videogular-themes-default"
+  },
+  "autoupdate": {
+    "source": "git",
+    "target": "git://github.com/videogular/bower-videogular-themes-default.git",
+    "basePath": "",
+    "files": [
+      "videogular*.+(css|map)",
```
Comment (PeterDaveHello) [inline ajax/libs/videogular-themes-default/package.json]: Why not just `"videogular*"`?

File: ajax/libs/videogular-themes-default/package.json [hunk 2]
```diff
@@ -0,0 +1,26 @@
...
+      "fonts/videogular.+(eot|svg|ttf|woff)"
```
Comment (PeterDaveHello) [inline ajax/libs/videogular-themes-default/package.json]: And `"fonts/**/*"`?

Comment (maruilian11) [inline ajax/libs/videogular-themes-default/package.json]: @PeterDaveHello do we use * in fonts in every lib?

Comment (PeterDaveHello) [inline ajax/libs/videogular-themes-default/package.json]: This is a release repo ... so why not?

**Comment by Amomo** [comment]
Hi @PeterDaveHello @maruilian11 
"videogular-themes-default" has a newer version now.
I had updated this PR, could you help me checking it again? thx~

**Comment by maruilian11** [comment]
@PeterDaveHello i think this PR is ok.

**Pull request merged**